hbase-common/pom.xml

```diff
@@ -281,6 +281,14 @@
         <groupId>net.revelc.code</groupId>
         <artifactId>warbucks-maven-plugin</artifactId>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-compiler-plugin</artifactId>
```

Comment:
why introduce it, and still use source 8?

Reply:
IDE miss import. Reverted the change